walkthroughs/week-5-tdd/project/src/main/webapp/script.js

```diff
@@ -24,14 +24,16 @@ function sendMeetingRequest() {
   // split it into an array of names
   const attendees = attendeesNamesString.split(/\s*,\s*/);
 
-// comma-separated list of names
-  const optionalAttendeesNamesString = document.getElementById('optional-attendees').value;
+  // comma-separated list of names
```

Comment:
How is this change related to the Pull Request?

Reply:
This was just a formatting change from running clang-format-path because the line was too long.